doc/start/documenting-ceph.rst

```diff
@@ -60,32 +60,33 @@ instead of compiling program source. This sequence (the sequence of building the
 Get the Source
 --------------
 
-Ceph documentation lives in the Ceph repository right alongside the Ceph source
-code under the ``ceph/doc`` directory. For details on github and Ceph,
-see :ref:`Get Involved`.
+The source of the Ceph documentation is a colletion of ReStructured Text files
+that are in the Ceph repository in the ``ceph/doc`` directory.  For details
+on github and Ceph, see :ref:`Get Involved`.
 
 The most common way to make contributions is to use the `Fork and Pull`_
 approach. You must:
 
-#. Install git locally. For Debian/Ubuntu, execute:
+#. Install git locally. In Debian or Ubuntu, run the following command:
 
    .. prompt:: bash $
 
 	sudo apt-get install git
 
-   For Fedora, execute:
+   In Fedora, run the following command:
 
    .. prompt:: bash $
 
 	sudo yum install git
 
-   For CentOS/RHEL, execute:
+   In CentOS/RHEL, run the following command:
 
    .. prompt:: bash $
 
 	sudo yum install git
 
-#. Ensure your ``.gitconfig`` file has your name and email address. :
+#. Make sure that your ``.gitconfig`` file has been configured to include your
```

Comment:
has been configured with your

Reply:
This is another matter where I think that the infinitival "to include" provides a disambiguating advantage over "with". I think that "with" phrases run the risk of being interpreted by the mind of the first-time reader as instrumental phrases (that is, phrases that explain just which thing you use to do another thing). So I'm going to let it stand, but out of respect for your view and for the open texture of language, I am going to make a note of it.